python/kafkaopmon/OpMonPublisher.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+import os
+import socket
+import inspect
+from opmonlib.opmon_entry_pb2 import OpMonValue, OpMonId, OpMonEntry 
+import google.protobuf.message as msg
+from google.protobuf.descriptor import FieldDescriptor as fd
+from google.protobuf.timestamp_pb2 import Timestamp
+from datetime import datetime
+from kafka import KafkaProducer
+from typing import Optional
+import logging
+
+class OpMonPublisher:
+    def __init__(
+                    self, 
+                    default_topic:str,
+                    bootstrap:str = "monkafka.cern.ch:30092", # Removed for if we don't want to use OpMon (i.e. for ssh-standalone)
+                    application_name:str = "python",
+                    package_name:str = "unknown"
```

Comment:
I just noticed that this is not used anywhere. What is the plan for this quantity? If there is none, please remove it.